app/src/ui/changes/changes-list.tsx

```diff
@@ -776,7 +776,10 @@ export class ChangesList extends React.Component<
           selectedRows={this.state.selectedRows}
           selectionMode="multi"
           onSelectionChanged={this.props.onFileSelectionChanged}
-          invalidationProps={this.props.workingDirectory}
+          invalidationProps={{
+            workingDirectory: this.props.workingDirectory,
+            isCommitting: this.props.isCommitting,
+          }}
```

Comment:
After the changes in this PR, I noticed the checkboxes of the files in the change list stayed disabled. Putting the `_refreshDirectory` and `refreshChangesSection` inside the `withIsCommitting` block meant that the `workingDirectory` prop would change *before* `isCommitting` is set to `false`. And when that happens, it wouldn't trigger a render on the list.